imagemounter/volume_system.py

```diff
@@ -73,6 +73,21 @@ def _make_subvolume(self, **args):
         self.volumes.append(v)
         return v
 
+    def _make_exfat_superfloppy(self):
```

Comment:
I have not been able to build an exFAT superfloppy that needs this method. A single volume could be mounted all of the times in my testing. Do you have an example where this is necessary?